omdata/orca/recipes.py

```diff
@@ -0,0 +1,132 @@
+import os
+import pickle
+import shutil
+
+from quacc.recipes.orca.core import ase_relax_job, static_job
+from quacc.wflow_tools.customizers import strip_decorator
+
+from omdata.orca.calc import (
+    OPT_PARAMETERS,
+    ORCA_BASIS,
+    ORCA_BLOCKS,
+    ORCA_FUNCTIONAL,
+    ORCA_SIMPLE_INPUT,
+    ORCA_SIMPLE_INPUT_QUACC_IGNORE,
+)
+
+
+def single_point_calculation(
+    atoms,
+    charge,
+    spin_multiplicity,
+    xc=ORCA_FUNCTIONAL,
+    basis=ORCA_BASIS,
+    orcasimpleinput=ORCA_SIMPLE_INPUT,
+    orcablocks=ORCA_BLOCKS,
+    nprocs=12,
+    outputdir=None,
+):
+    """
+    Wrapper around QUACC's static job to standardize single-point calculations.
+    See github.com/Quantum-Accelerators/quacc/blob/main/src/quacc/recipes/orca/core.py#L22
+    for more details.
+
+    Arguments
+    ---------
+
+    atoms: Atoms
+        Atoms object
+    charge: int
+        Charge of system
+    spin_multiplicity: int
+        Multiplicity of the system
+    xc: str
+        Exchange-correlaction functional
+    basis: str
+        Basis set
+    orcasimpleinput: list
+        List of `orcasimpleinput` settings for the calculator
+    orcablocks: list
+        List of `orcablocks` swaps for the calculator
+    nprocs: int
+        Number of processes to parallelize across
+    outputdir: str
+        Directory to move results to upon completion
+    """
+
+    o = strip_decorator(static_job)(
+        atoms,
+        charge=charge,
+        spin_multiplicity=spin_multiplicity,
+        xc=xc,
+        basis=basis,
+        orcasimpleinput=orcasimpleinput + ORCA_SIMPLE_INPUT_QUACC_IGNORE,
+        orcablocks=orcablocks,
+        nprocs=nprocs,
+    )
+    # TODO: how we want to handle what results to save out and where to store
+    # them.
+    os.makedirs(outputdir, exist_ok=True)
+    shutil.move(o["dir_name"], outputdir)
+    with open(os.path.join(outputdir, "quacc_results.pkl"), "wb") as f:
+        pickle.dump(o, f)
```

Comment:
You can probably do 

```
from quacc import SETTINGS
SETTINGS.RESULTS_DIR = outputdir
```

to modify the global variable in-place.

Reply:
Thanks! I knew there had to be an easy way to do it directly in quacc :)

Reply:
`quacc_results.pkl` here corresponds to the `cclib_summarize_run` output, which is just a dictionary so this works as intended. The orca output files that live in the `quacc-XXXX` directory that gets created however are indeed all `.gz`.

Reply:
Ah, to clarify, that behavior was a bug fixed in https://github.com/Quantum-Accelerators/quacc/pull/1895. If `GZIP_FILES` is enabled, which it is by default, the output of `cclib_summarize_run` is a `.pkl.gz` and not `.pkl`. This is as of PR #1895.

Reply:
Ah got ya, okay I'll update then.